WebApp-OpenIDConnect-DotNet/Views/Shared/_LoginPartial.cshtml

```diff
@@ -2,6 +2,8 @@
 {
     <ul class="nav navbar-nav navbar-right">
         <li class="navbar-text">Hello @User.Identity.Name!</li>
+        <li><a asp-area="" asp-controller="Session" asp-action="EditProfile">Edit Profile</a></li>
+        <li><a asp-area="" asp-controller="Session" asp-action="ResetPassword">Reset Password</a></li>
```

Comment:
I looked a bit at the B2C UI customization support and I see how customizing the profile editing experience could be done, but it looks like it requires an addressable endpoint to get the UI customization, so I think what you have here is fine. Alternative we could have a separate profile page in the template linked to from the displayed user name that lists the edit profile and reset password links.